sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_receipts_async.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_recognize_receipts_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze receipts from both a file and a url.
+
+USAGE:
+    python sample_recognize_receipts_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+"""
+
+import os
+import asyncio
+
+
+class RecognizeReceiptsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    async def recognize_receipts_from_file(self):
+        # TODO: this can be used as examples in sphinx
+        print("=========Recognize Receipts from a file=========")
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+        form_recognizer_client = FormRecognizerClient(endpoint=self.endpoint, credential=AzureKeyCredential(self.key))
+        with open("../sample_forms/receipt/contoso-allinone.jpg", "rb") as f:
+            receipts = await form_recognizer_client.begin_recognize_receipts(stream=f.read())
+
+        for idx, receipt in enumerate(receipts):
+            print("--------Recognizing receipt #{}--------".format(idx))
+            print("Total: {} (Confidence score of {}, based on text value '{}', with bounding box {})".format(
```

Comment:
I'm thinking we should include more receipt fields here. i'm almost wondering if we should have two samples here, one that is more value centric -- getting the values from the receipt and the other more layout centric -- getting bounding boxes, raw ocr text_content. what do you think?

Reply:
That would definitely be neater, but the way Paul described it was that these tended to be used in tandem. Actually reading the email more, we could have a validate_information sample?

"For these customers, they need the confidence for each extracted field value (top feature ask) to highlight the ones needing validation.  While most of them ultimately want the normalized semantic value for each field (ex. ISO date instead of raw date string), most still need the corresponding text value to simplify manual validation.  Further, to help human validators check the results, they need to access the bounding box associated with each field value to highlight the location of the extracted field.  (Note that page number is encoded in the element reference path, not in the referenced word itself.)"